README.adoc

```diff
@@ -31,12 +35,12 @@ There are some limitations at the moment.
 This extension is built against Neo4j's `openCypher-frontend` and uses specific _3.4_ packages.
 It may work with any other _3.1+_ version of Neo4j in server mode.
 
-If you are using the extension with Neo4j running embedded, it might end in version conflicts regarding Scala, at least.
-There will be multiple Scala runtimes in your classpath.
+The current implementation has to override a lot of existing methods to generate the string representation of the manipulated Cypher statement.
+That might break things in newer versions of openCypher.
 
+If you are using this extension with an embedded Neo4j instance, it might end up in version conflicts regarding Scala, as there will be multiple Scala runtimes in your class path.
 
 ==== Limited keywords
-The implementation overrides a lot of existing methods to generate the string representation of the manipulated Cypher statement.
```

Comment:
Maybe something like
"The API used doesn't provide means for generating a string representation of all Cypher parts, so we might not support correct rewriting of all valid statements."